code/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
```

Comment:
Can you add the license plugin here? 
apply(plugin = "aep-license")

Reply:
Adding the plugin here actually made no difference. I see user profile don't have them too 
https://github.com/adobe/aepsdk-userprofile-android/blob/main/code/build.gradle.kts